.travis.yml

```diff
@@ -53,8 +44,12 @@ env:
   - TARGET=":jsettlers.logic:test:aiDifficultiesIT"
   - TARGET=":jsettlers.logic:test:autoReplayIT"
   - TARGET=":jsettlers.logic:test:replayValidationIT"
-    
-  # Test release generation
+  
   - TARGET="release"
 
+  
 script: GRADLE_OPTS='-Xmx600m -Dorg.gradle.jvmargs="-Xmx1500m"' ./gradlew $TARGET
+
+
+after_success:
+- ./deploy.sh
```

Comment:
You're right, I misunderstood the documentation... I will have a look on changing it to  a gradle task.